application/templates/cms/edit_measure_version.html

```diff
@@ -211,6 +193,73 @@ <h3 class="govuk-heading-m">Updates and corrections</h3>
         </div>
     </form>
 
+    <div class="govuk-grid-row govuk-!-margin-bottom-5">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-form-group {% if data_sources_not_added %}govuk-form-group--error{% endif %}">
+          <h2 class="govuk-heading-m" id="data-sources">Data sources</h2>
+          <div class="govuk-!-margin-bottom-5">
+            {% if new %}
+              <p class="govuk-body">Once this page is saved you will be able to add data sources</p>
+            {% else %}
+              <h3 class="govuk-heading-s">Primary data source</h3>
+
+              {% if measure_version.primary_data_source %}
+                <div class="govuk-grid-row">
+                  <div class="govuk-grid-column-two-thirds">
+                    <p class="govuk-body">{{ measure_version.primary_data_source.title }}</p>
+                  </div>
+                  <div class="govuk-grid-column-one-half">
+                    <p class="govuk-body">
+                      <a class="govuk-link" href="{{ url_for('cms.edit_data_source', topic_slug=topic.slug, subtopic_slug=subtopic.slug, measure_slug=measure.slug, version=measure_version.version, data_source_id=measure_version.primary_data_source.id) }}">Make changes to this <span class="govuk-visually-hidden">primary</span> data source</a>
+                    </p>
+                  </div>
+                  <div class="govuk-grid-column-one-half">
+                    <form method="POST" action="{{ url_for('cms.remove_data_source', topic_slug=topic.slug, subtopic_slug=subtopic.slug, measure_slug=measure.slug, version=measure_version.version, data_source_id=measure_version.primary_data_source.id) }}">
+                      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+
+                      <button type="submit" class="govuk-!-font-size-19 govuk-button govuk-link eff-button-link eff-button-link--destructive" href="{{ url_for('cms.remove_data_source', topic_slug=topic.slug, subtopic_slug=subtopic.slug, measure_slug=measure.slug, version=measure_version.version, data_source_id=measure_version.primary_data_source.id) }}">Remove<span class="govuk-visually-hidden"> primary data source</span></button>
```

Comment:
Feels a bit weird to include both `govuk-button` and `govuk-link` – can we drop `govuk-link` and move any required styles to `eff-button-link`?

Should also be `eff-button--link` now (two dashes).

Reply:
> Feels a bit weird to include both `govuk-button` and `govuk-link` – can we drop `govuk-link` and move any required styles to `eff-button-link`?

Done.

> Should also be `eff-button--link` now (two dashes).

As in a separate comment reply, I've gone with saying that `eff-button-link` is an entire standalone component. This can then have modifiers, such as `warning`, to create `eff-button-link--warning`. I think this is the right way to go about it.